README.md

```diff
@@ -1,57 +1,97 @@
 # Website Digital Product School
 
-# Prerequesites
+# Frontend
 
-## Node and npm
+## Prerequesites
 
-` Install node and npm `
+### Node and npm
 
-## Gatsby
+`Install node and npm`
+
+### Gatsby
 
 `npm install -g gatsby-cli`
 
+## Context
+
+The gatsby frontend resides currently in the root directory. The intention is to move it to e `frontend/` folder.
+
+## Setting Up
+
+1. Run `npm install` in the root dir
+
+## Development
+
+1. Run `npm run start`
+2. Any environemnt variables go into `.env.development` for local development and into `cloudbuild-prod|staging.yaml` for prod / staging. This is a bit inconsistent, it might be best to move those from `cloudbuild-prod|staging.yaml` in `.env.staging` and `.env.production`, but yolo unless you wanna fix it.
```

Comment:
The documentation of Gatsby suggest not to bring `.env.*` into a repository.
https://www.gatsbyjs.org/docs/environment-variables/#example

I'd rather keep it "inconsistent" for now, as we have the option of encryption in the cloudbuild.yaml files.